com.endlessm.HackToolbox.json.in

```diff
@@ -43,6 +43,20 @@
         }
       ]
     },
+    {
+      "name": "Hack-font",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D -m644 *.ttf -t /app/share/fonts"
```

Comment:
lesson learn from knowledge apps, we need to `"post-install": ["fc-cache -f /app/share/fonts/`"]`

Reply:
huh, it worked without that! But I'll add it